File: 1672-richest-customer-wealth/1672-richest-customer-wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
     let max = 0,sum;
        for(let i=0;i<accounts.length;i++){
            sum = 0;
            for(let j=0;j<accounts[0].length;j++){
                sum+=accounts[i][j];
            }
            if(max<sum) max=sum;
        }
        return max;
};

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int max = 0,sum;
        for(int i=0;i<accounts.size();i++){
            sum = 0;
            for(int j=0;j<accounts[i].size();j++){
                sum+=accounts[i][j];
            }
            if(max<sum) max=sum;
        }
        return max;
    }
};